SQL/Archive/20.0/2018-01-24-ScanTypeExcludeInConfig.sql

```diff
@@ -0,0 +1,7 @@
+-- Insert into ConfigSettings scan_type_exclude
+INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple,
+DataType, Parent, Label, OrderNumber) SELECT 'excluded_series_description', 'Series description to to be excluded from insertion into the database (typically localizers and scouts)', 1, 1, 'text', ID, 'Series description to exclude from imaging insertion', 18 FROM ConfigSettings WHERE Name="imaging_pipeline";
```

Comment:
@cmadjar 
a small thing I just noticed as I am tetsing your MRI one;
there is a typo here 'to to' should be 'to'

Reply:
@MounaSafiHarab this will be fixed once #3550 is merged